RadixWallet/Core/SharedModels/Entities/EntitiesControlledByFactorSource.swift

```diff
@@ -96,35 +98,11 @@ extension EntitiesControlledByFactorSource {
 		deviceFactorSource.isExplicitMain
 	}
 
-	public var factorSourceID: FactorSourceID.FromHash {
+	public var factorSourceID: FactorSourceIDFromHash {
 		deviceFactorSource.id
 	}
 
-	public var mnemonicWordCount: BIP39.WordCount {
+	public var mnemonicWordCount: BIP39WordCount {
 		deviceFactorSource.hint.mnemonicWordCount
 	}
 }
-
-extension DeviceFactorSource {
-	/// **B**abylon **D**evice **F**actor **S**ource
-	public var isExplicitMainBDFS: Bool {
```

Comment:
moved to https://github.com/radixdlt/babylon-wallet-ios/blob/sargon_profile_integration_stage_1_batch_1/RadixWallet/ProfileSargonMigration/Stage2/Atoms/Stage2MigrateToSargon%2BDeviceFactorSource.swift#L14